README.md

```diff
@@ -19,9 +19,19 @@ we recommend to use the most recent stable version of the rust compiler.
 
 Note:
 
-* on windows, you'll need to add the `/userProfile/.cargo/bin` into the Path;
-* on linux and OSX: add `${HOME}/.cargo/bin` to your `${PATH}`
-* on linux with systemd: to enable logging to journald replace step 5. with `cargo install --path . --features systemd`
+* On Windows, you'll need to add the `%USERPROFILE%\.cargo\bin` into the
+  environment variable `PATH`.
+* On Linux and macOS: add `${HOME}/.cargo/bin` into your `PATH`.
+* Make sure the C compiler toolchain is installed and, on Unix (e.g. macOS),
+  the compiler and linker executable `cc` is found in `PATH`.
+* On Linux with systemd: to enable logging to journald replace step 6
+  with `cargo install --path . --features systemd`.
+* On Linux environments without glibc, such as NixOS or Alpine, the protobuf
```

Comment:
On NixOS you shouldn't install such tools manually but rely on `shell.nix` (which sets `PROTOC`).

Reply:
Thank you, I am chastised :bowing_man: Fixed in 00c0064.